### DISCUSSION TITLE
Table/View definitions not found in new supabase

### DISCUSSION THREAD

**Discussion by msafrun** [supabase/supabase#21310] [General]
Hi I notice that Table/View definitions not found in new supabase. Will this feature discontinued, or the team is still on progress for new migration? Because the feature is really helpful, I hope it will still be continued...

* before
![Screenshot 2024-02-16 142525](https://github.com/supabase/supabase/assets/103011851/3326618b-886a-4d3a-929b-f49394f51d49)

* now
![Screenshot 2024-02-16 142734](https://github.com/supabase/supabase/assets/103011851/6e59e926-7a49-4224-a288-5deaa62ecbcb)

**Comment by sacsand** [comment]
really missing this feature , Very useful while updating views . 

Update :: moved to  bottom right corner 
![image](https://github.com/supabase/supabase/assets/9622671/0032cdd3-0bbb-4f38-9b28-36d244f5a43f)
